Hermes/appscale/hermes/stats/producers/tests/test_cluster_stats.py

```diff
@@ -126,6 +126,27 @@ def test_failure_of_node(self, mock_get_current, mock_fetch,
     self.assertEqual(local_stats.utc_timestamp, 1494248091.0)
     self.assertEqual(failures, {'192.168.33.11': '500 Timeout error'})
 
+  @patch.object(cluster_stats.appscale_info, 'get_private_ip')
+  @patch.object(cluster_stats.ClusterNodesStatsSource, 'ips_getter')
+  @patch.object(node_stats.NodeStatsSource, 'get_current')
+  @testing.gen_test
+  def test_local_failure(self, mock_get_current, mock_ips_getter,
+                         mock_get_private_ip):
+    # Mock appscale_info functions for getting IPs
+    mock_get_private_ip.return_value = '192.168.33.10'
+    mock_ips_getter.return_value = ['192.168.33.10']
+    # Mock local source
+    mock_get_current.side_effect = ValueError(u"Something strange \u2234")
```

Comment:
Is that the 'therefore' character?

Reply:
yes) Don't try to find reason why I chose this symbol.